src/lib.rs

```diff
@@ -399,34 +410,39 @@ impl<'a> Exporter<'a> {
         let mut context = Context::new(src.to_path_buf(), dest.to_path_buf());
 
         let (frontmatter, mut markdown_events) = self.parse_obsidian_note(src, &context)?;
-        context.frontmatter = frontmatter;
-        for func in &self.postprocessors {
-            match func(&mut context, &mut markdown_events) {
-                PostprocessorResult::StopHere => break,
-                PostprocessorResult::StopAndSkipNote => return Ok(()),
-                PostprocessorResult::Continue => (),
-            }
-        }
+        match frontmatter.get(self.ignore_frontmatter_keyword) {
+            Some(serde_yaml::Value::Bool(true)) => Ok(()),
```

Comment:
this match is the meat of the change - the bits below are just the other branch of that case being indented - no change in functionality there

Reply:
turning off whitespace changes helps with the review here